packages/test-support/src/project.ts

```diff
@@ -200,8 +204,6 @@ export function getEmber4AppProject(): Project {
 }
 
 export async function setupProject(project: Project): Promise<Project> {
-  const { name: tmpDir } = tmp.dirSync();
-  project.baseDir = tmpDir;
```

Comment:
You don't need to create your own tmp dir. Fixturify-project generates one for you when you `write()`